main.go

```diff
@@ -48,13 +54,19 @@ var (
 
 var c Specification
 
+type SessionData struct {
+	Email string
+	// Add other fields as needed
+}
+
 // Specification is the config struct
 type Specification struct {
 	Port           string `envconfig:"PORT" required:"true"`
 	CaptchaSitekey string `required:"true"`
 	CaptchaSecret  string `required:"true"`
 	SlackToken     string `required:"true"`
 	CocUrl         string `required:"false" default:"http://coc.golangbridge.org/"`
+	SessionData    []byte
```

Comment:
Protip: if this is JSON use `json.RawMessage` to prevent encoding errors.

Reply:
thanks for the tip!